products/express.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Express
+category: framework
+tags: javascript herodevs
+iconSlug: express
+permalink: /express
+releasePolicyLink: https://expressjs.com/en/support/
+changelogTemplate: https://github.com/expressjs/express/releases/tag/__LATEST__
```

Comment:
This may be an issue for some versions, such as https://github.com/expressjs/express/releases/tag/v5.0.0 which has a `v` prefix. Given there is no prefix for 5.0.1 it may have been a typo, so I suggest we do nothing for now.